.github/workflows/reusable-run-go-acceptance-test.yml

```diff
@@ -0,0 +1,40 @@
+name: test-run-go-tests-for-path
+
+on:
+  workflow_call:
+    inputs:
+      name:
+        description: 'The name to use that will appear in the output log file artifact'
+        required: true
+        type: string
+      path:
+        description: 'The path to the test without the precedeing "./" or following "/..." e.g. go test -v ./$path/...'
+        required: true
+        type: string
+    # We will need to add the capacity for receiving passed secrets once we get to the tests that require API credentials
+
+jobs:
+  product-metadata:
```

Comment:
Since there's only one job, other than this, do we really need this to be a separate job? I'm okay either way, but I'm pretty sure this workflow would be slightly faster if it only has one job.